app/src/ui/toolbar/dropdown.tsx

```diff
@@ -75,6 +75,9 @@ export interface IToolbarDropdownProps {
   /** The button's style. Defaults to `ToolbarButtonStyle.Standard`. */
   readonly style?: ToolbarButtonStyle
 
+  /** Wether the dropdown will trap focus or not. Defaults to true. */
+  readonly focusTrapActive?: boolean
```

Comment:
I'm open to suggestions for better names 😂

Reply:
We typically use `enableX` or `disableX`, see https://github.com/desktop/desktop/blob/1c488e94f56671535ea5de2dbcccd5d13d0c434f/app/src/ui/app-menu/app-menu.tsx#L27-L32 for an example.

So probably
```suggestion
  readonly enableFocusTrap?: boolean
```